docs/README.md

```diff
@@ -0,0 +1,124 @@
+# Bazel Central Registry (BCR) contribution guidelines
+
+Before contributing to the Bazel Central Registry, check the [BCR policies](bcr-policies.md) to learn how the BCR is maintained.
+
+## Contribute a Bazel module
+
+To contribute a new module or a new version to an existing module, you can clone the BCR repository and run the interactive helper script:
+
+```bash
+git clone https://github.com/bazelbuild/bazel-central-registry.git
+cd bazel-central-registry
+python3 ./tools/add_module.py
+```
+
+The script will generate all require changes based on your input, please review, modify and commit the change, then send a PR to the BCR repository.
+
+If you are the project owner, you can set up the [Publish to BCR](https://github.com/apps/publish-to-bcr) Github App for your repository to automatically send a PR to the BCR when cutting a new release.
+
+## Presubmit
+
+Every module version must pass the BCR presubmit before getting merged. The presubmit validate the correctness and consistency of module information, then runs build and test targets specified in the `presubmit.yml` file. The BCR presubmit is driven by the [bcr_presubmit.py](https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazel-central-registry/bcr_presubmit.py) script on [Bazel CI](https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#bazel-continuous-integration).
```

Comment:
the presubmit validate -> the presubmit validate**s**

Reply:
Done